Module deployments fail on CF

Creating a ticktock stream the module deployments fail with:

```
2016-03-03T16:55:26.000+00:00 [APP] ERR Caused by: org.springframework.context.ApplicationContextException: Failed to start bean 'outputBindingLifecycle'; nested exception is org.springframework.context.ApplicationContextException: Failed to start bean 'inputBindingLifecycle'; nested exception is org.springframework.data.redis.RedisConnectionFailureException: Cannot get Jedis connection; nested exception is redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
```

--- a/spring-cloud-dataflow-deployer-cloudfoundry/src/main/java/org/springframework/cloud/dataflow/module/deployer/cloudfoundry/CloudFoundryModuleDeployerProperties.java
+++ b/spring-cloud-dataflow-deployer-cloudfoundry/src/main/java/org/springframework/cloud/dataflow/module/deployer/cloudfoundry/CloudFoundryModuleDeployerProperties.java
@@ -88,7 +88,7 @@ class CloudFoundryModuleDeployerProperties {
 	/**
 	 * The buildpack to use for deploying the application.
 	 */
-	private String buildpack = "https://github.com/cloudfoundry/java-buildpack.git#master";
+	private String buildpack = "https://github.com/cloudfoundry/java-buildpack.git#v3.6";
 
 	/**
 	 * The amount of memory (MB) to allocate, if not overridden per-module.

